### PR TITLE
Disable light mode for league tables, and update requirements.txt file

### PR DESCRIPTION
**Pull request by ash663**
- discord.py > 1.7.3 introduces breaking changes, so I am adding specific versions of the packages to use to avoid confusion.

- There were a couple of missing packages from the file too, added them (lxml, fotmob).

- Disabling light mode since it's ugly, and the only user of it was banned (Blair)

Screenshots where I tested the changes: https://discord.com/channels/193393364819771392/617026340297900032/1093327968488652830

### CODE REVIEW

File: cogs/tables.py [hunk 1]
```diff
@@ -25,8 +25,11 @@ async def leagueTable(self, ctx,
         # Check if (w)hite or (l)ight, else use dark mode
```
Comment (ash663) [inline cogs/tables.py:25]: I'll update the comment in a future commit

**Review by reddit-tennis — APPROVED** [review]
LGTM.

We need to move to Poetry soon (tm)

File: cogs/tables.py [hunk 2]
```diff
@@ -25,8 +25,11 @@ async def leagueTable(self, ctx,
...
         light_mode = background.startswith('l') or background.startswith('w')
 
-        bg_colour = (255, 255, 255) if light_mode else (47, 49, 54)
-        font_colour = (0, 0, 0) if light_mode else (255, 255, 255)
+        if (light_mode):
```
Comment (reddit-tennis) [inline cogs/tables.py:28]: Not sure why the brackets here. Anyways in the future we will introduce Black or similar so we all write similar styles :)

**Review by ingenium21 — APPROVED** [review]
i enjoy the new light mode response. 10/10 would approve

**Pull request merged**